CHANGELOG.md

```diff
@@ -18,6 +18,10 @@
 
 #### Enhancements
 
+* Add `no_constant_condition` rule which warns when a `if`, `guard` or `switch` condition is a constant value that would never change at runtime.  
```

Comment:
This should be wrapped like the other entries

Reply:
Done